node-express/accelerator.axl

```diff
@@ -0,0 +1,38 @@
+engine {
+
+  applyTo("README.md") {
+    TextPreprocessor()
+  }
+
+  if (#deploymentType == 'none') {
+    select(!"deploy/**")
```

Comment:
Small nitpick, but can't this be pulled out of that (then unnecessary) `if` and put at the end?

Reply:
Yep, made that change